README.md

```diff
@@ -65,11 +65,14 @@ If you make changes to the template you can create a test site with this command
 Make sure that you've committed your changes first, because Middleman works with Git.
```

Comment:
I know this isn't part of the PR, but is this true? Seems to serve up my uncommitted changes just fine...

Reply:
Are you talking about serving up changes to documentation? Because this documentation is about making changes to the template – when running `middleman init -T file://$(pwd)` it'll use the git index at that location rather than just copying the files directly.

Reply:
Ahhhh, I see, `-T` takes a git repository, and we're using the file protocol to test changes locally. Gotcha.